src/credential/issuance/07-verify-and-parse-credential.ts

```diff
@@ -30,6 +30,8 @@ export type VerifyAndParseCredential = (
   }
 ) => Promise<{
   parsedCredential: ParsedCredential;
+  credential: Out<ObtainCredential>["credential"];
```

Comment:
As above, why do we return `credential` and `credentialConfigurationId` if they are inputs of the function?